plugins/ltac/tauto.ml

```diff
@@ -242,7 +241,7 @@ let tauto_uniform_unit_flags = {
 }
 
 (* This is the compatibility mode (not used) *)
-let tauto_legacy_flags = {
+let _tauto_legacy_flags = {
```

Comment:
It was introduced in e1f5180e88bf02a22c954ddbdcbdfeb168d264a6 (@herbelin) and was already not used at that time.
I don't know.